Verslag/methodology.tex

```diff
@@ -22,11 +22,11 @@ \subsection{Network analysis}
 
 \subsection{Code analysis}
 
-For the code analysis, JadX and Dex2Jar were used to decompile the application. This created the opportunity to read the code. An attempt was made to analyze the code. Analyzing the code was very difficult due to obfuscation. Where possible it was documented whether the code was malicious or not.
+For the code analysis, JadX and Dex2Jar were used to decompile the application. Both of these applications were used since it is useful to review the code using both of the applications. This would allow us to double check the code and make sure the decompiled APK's code is accurate. Analyzing the code was very difficult due to obfuscation. Where possible it was documented whether the code was malicious or not.
```

Comment:
Both of these applications were used to review the code. This would allow us to double check the code and make sure the decompiled APK's code is accurate.